index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- font CDN -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!-- HEADER SECTION -->
+    <div class="header">
+        <div class="header__logo">David</div>
+        <div class="header__menu-btn">
+            <div class="menu-btn__ham"></div>
+        </div>
+    </div>
+    <!-- HEADLINE SECTION -->
+    <main>
+        <section class="headline-section">
+            <h1 class="headline-section__title">I'm David<br/>Glad to see you!</h1>
+            <p class="headline-section__desc">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <h2 class="headline-section__sub-title">
+                LET’S CONNECT
+            </h2>
+            <div class="headline-section__social">
+                <a href="#" class="social-icon twitter"><img src="images/twitter.svg" alt="twitter icon"></a>
+                <a href="#" class="social-icon linkedin"><img src="images/Linkedin.svg" alt="Linkedin icon"></a>
+                <a href="#" class="social-icon linkedin"><img src="images/medium.svg" alt="medium icon"></a>
+                <a href="#" class="social-icon github"><img src="images/github.svg" alt="github icon"></a>
+                <a href="#" class="social-icon chill"><img src="images/chill.svg" alt="chill icon"></a>
+            </div>
```

Comment:
- [ ] Good job adding the social icons in an `a` tag 🔥. Please consider using a `ul` tag to wrap all the social icons and use `li` tag to wrap each of the social icons instead of a `div` tag. This is to improve the semantic structure of your page and improve the code structure too. See my suggestion below:

```suggestion
            <ul class="headline-section__social">
                <li>
                  <a href="#" class="social-icon twitter"><img src="images/twitter.svg" alt="twitter icon"></a>
                </li>
                <li>
                  <a href="#" class="social-icon linkedin"><img src="images/Linkedin.svg" alt="Linkedin icon"></a>
                </li>
                <li>
                  <a href="#" class="social-icon linkedin"><img src="images/medium.svg" alt="medium icon"></a>
                </li>
                <li>
                  <a href="#" class="social-icon github"><img src="images/github.svg" alt="github icon"></a>
                </li>
                <li>
                  <a href="#" class="social-icon chill"><img src="images/chill.svg" alt="chill icon"></a>
                </li>
            </ul>
```